thunder.profile

```diff
@@ -239,12 +239,62 @@ function thunder_themes_installed($theme_list) {
   }
 }
 
+/**
+ * Check if provided triggering modules are one of just installed modules.
+ *
+ * This function is helper for thunder_modules_installed.
+ *
+ * @param array $modules
+ *   The list of the modules that were newly installed.
+ * @param array $triggering_modules
+ *   The list of triggering modules required for executing some action.
+ * @param bool $only_module_enable
+ *   Support only module installations done by module enabling. This flag will
+ *   exclude checks for module installation when profile is installed or when
+ *   module installation is triggered by configuration import.
+ *
+ * @return bool
+ *   Returns if triggering module is newly installed.
+ */
+function _thunder_check_triggering_modules(array $modules, array $triggering_modules, $only_module_enable = TRUE) {
+  // Sometimes we want to react only on module installations done by enabling of
+  // module. In that case we want to ignore module installation when profile is
+  // installed or when module is enabled over configuration import.
+  if (
+    $only_module_enable
+    && (drupal_installation_attempted() || Drupal::isConfigSyncing())
+  ) {
+    return FALSE;
+  }
+
+  // Check that at least one triggering module is in list of the modules that
+  // were newly installed.
+  $triggering_not_installed_modules = array_diff($triggering_modules, $modules);
+  if (count($triggering_not_installed_modules) === count($triggering_modules)) {
+    return FALSE;
+  }
+
+  // All required triggering modules are in the list of the modules that were
+  // newly installed.
+  if (empty($triggering_not_installed_modules)) {
+    return TRUE;
+  }
```

Comment:
We need `$modules` because we have to check if "hook_modules_installed" is triggered because of one of the modules we want to react on.

`if (empty($triggering_not_installed_modules)) {` is there only for performance reasons. I can kill it, but then we have to do `getModuleList`, `array_keys` and `array_diff`.